### PR TITLE
fix: privacy field on ctl

### PR DESCRIPTION
**Pull request by psanders**
## Description

This PR is needed to fix the CTL not sending privacy settings. 

Also:

- [x] Re-enabled test, lint, check-types, and format on pre-commit. 
- [x] Fixed linting warnings
- [x] Added copyright notice to the `ctl` and `voice` modules 

## Type of change

Bug fix + Code cleanup.

## How Has This Been Tested?

Manually tested all changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by psanders** [comment]
@efraa I recommend you review this by commit. Not need to review d4ff536d04fc14485823bbec2651b02fa93f1657 since it a linting commit.

File: docs/reference/VoiceResponse.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ first setup a TTS plugin such as MaryTTS, GoogleTTS, or AWS PollyTTS
 async function handler (request, response) {
   await response.answer();
   response.use(new GoogleTTS())
-  await response.say("Hello world");   // Plays the sound using GoogleTTS's default values
+  await response.say("Hello workd");   // Plays the sound using GoogleTTS's default values
```
Comment (efraa) [inline docs/reference/VoiceResponse.md]:
```suggestion
  await response.say("Hello world"); // Plays the sound using GoogleTTS's default values
```

File: docs/reference/VoiceResponse.md [hunk 2]
```diff
@@ -130,13 +130,13 @@ Waits for data entry from the user's keypad or from a speech provider.
 | options.numDigits | <code>number</code> | Milliseconds to skip before playing. Only applies to the first URI if multiple media URIs are specified |
 | options.timeout | <code>number</code> | Milliseconds to wait before timeout. Defaults to 4000. Use zero for no timeout. |
 | options.finishOnKey | <code>string</code> | Optional last character to wait for. Defaults to '#'. It will not be included in the returned digits |
-| options.source | <code>string</code> | Where to listen as input source. This option accepts `dtmf` and `speech`. A speech provider must be configure when including the `speech` source. You might include both with `dtmf,speech`. Defaults to `dtmf` |
+| options.source | <code>string</code> | Where to listen as input source. This option accepts `dtmf` and `speech`. A speech provider must be configure when including the `speech` source. You might inclue both with `dtmf,speech`. Defaults to `dtmf` |
```
Comment (efraa) [inline docs/reference/VoiceResponse.md]:
```suggestion
| options.source | <code>string</code> | Where to listen as input source. This option accepts `dtmf` and `speech`. A speech provider must be configure when including the `speech` source. You might include both with `dtmf,speech`. Defaults to `dtmf` |
```

File: docs/reference/VoiceResponse.md [hunk 3]
```diff
@@ -317,7 +317,7 @@ async function handler (request, response) {
 
 ### voiceResponse.answer()
 Answer the communication channel. Before running any other verb you
-must run the answer command.
+must run the anwer command.
```
Comment (efraa) [inline docs/reference/VoiceResponse.md]:
```suggestion
must run the answer command.
```

**Review by efraa — DISMISSED** [review]

**Pull request merged**